hivemind/p2p/p2p_daemon_bindings/p2pclient.py

```diff
@@ -61,8 +61,8 @@ async def listen(self) -> AsyncIterator["Client"]:
         async with self.control.listen():
             yield self
 
-    async def add_unary_handler(self, proto: str, handler: TUnaryHandler):
-        await self.control.add_unary_handler(proto, handler)
+    async def add_unary_handler(self, proto: str, handler: TUnaryHandler, balanced: bool = False):
+        await self.control.add_unary_handler(proto, handler, balanced=balanced)
```

Comment:
Perhaps I couldn't find it, but can you please show how the `balanced` argument is eventually used? Right now I can only see that it is passed around the code with no obvious reason